language/reserved_keywords.rb

```diff
@@ -0,0 +1,159 @@
+require_relative '../spec_helper'
+
+describe "Ruby's reserved keywords" do
+  # Copied from Prism::Translation::Ripper
+  keywords = %w[
+    alias
+    and
+    begin
+    BEGIN
+    break
+    case
+    class
+    def
+    defined?
+    do
+    else
+    elsif
+    end
+    END
+    ensure
+    false
+    for
+    if
+    in
+    module
+    next
+    nil
+    not
+    or
+    redo
+    rescue
+    retry
+    return
+    self
+    super
+    then
+    true
+    undef
+    unless
+    until
+    when
+    while
+    yield
+    __ENCODING__
+    __FILE__
+    __LINE__
+  ]
+
+  keywords.each do |kw|
```

Comment:
```suggestion
  keywords.each do |name|
```
`kw` feels too confusing here, especially since there is a spec about keyword arguments below.
Since we are mostly testing names for variables/methods, `name` seems clearer in usages.

Reply:
Mmmmm I disagree, in a spec focused on describing the behaviour of keywords, I would expect "keyword" to a be prominent term. It's an overloaded term, but that's just how Ruby is already. Perhaps `keyword` is better than `kw`?

Anyway, I renamed to `name`